src/esp/gfx/GenericDrawable.cpp

```diff
@@ -147,10 +150,16 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
     shader_->bindNormalTexture(*(materialData_->normalTexture));
   }
 
-  shader_->draw(mesh_);
+  shader_->draw(getMesh());
```

Comment:
I looked into the `getMesh()` which has a sanity check there. Great.
However, better solution is to add such sanity check at the beginning of this function, and make it fail even faster (without even setting the shader, because it does not make any sense.)

Reply:
For the "sanity check" here, I blocked `draw` when `isRendererCreated==false` instead of having an assert. This goes along with Eric's comments about minimal invasiveness. Otherwise I would have to block every draw call elsewhere in the code.

Reply:
My comments will not require you to do massive refactoring. So it aligns with your design rules -- minimal changes. 
>I would have to block every draw call elsewhere in the code.

You misunderstood "the beginning of this function". I did not ask you to add sanity check everywhere when you call this function. 
I am asking you to add sanity check at the start of this function definition. 
(Line 119 before updateShader()).

Reply:
Sorry, I should've been more clear. I meant that I added
```if (!isRendererCreated) {
    return;
  }```
before `updateShader()`, rather than an assert statement (which is what I think you meant by sanity check). I have not pushed this change yet.